packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -357,7 +357,7 @@ export function SelectBase<T>({
             return <DropdownIndicator isOpen={props.selectProps.menuIsOpen} />;
           },
           SingleValue(props: any) {
-            return <SingleValue {...props} disabled={disabled} />;
+            return <SingleValue {...props} isDisabled={disabled} />;
```

Comment:
any reason why this was changed?

Reply:
Yep, `isDisabled` is the prop that is destructured in `SingleValue` to check if the option should be disabled – however we've always been passing `disabled` in to the component instead of `isDisabled`.

The functionality to disable select options seems to have been broken for quite a while.